src/spac/data_utils.py

```diff
@@ -305,8 +305,9 @@ def load_csv_files(file_names):
         A list of pandas dataframe of all the csv files.
     """
 
-    meta_schema = []

```

Comment:
@ruiheesi , would you please mention that returned dataframe will have an extra column called "loaded_file_name" added?